### PR TITLE
feat: store business info in firestore and business logo in storage

### PR DESCRIPTION
**Pull request by thetimothyp**
This PR stores submitted business info in Firestore and Firebase Storage (for images).

(The image from the gif and the screenshot don't match up because I recorded separately from taking the screenshot)

business-info-firestore

image

image

### CODE REVIEW

File: src/firebase.js [hunk 1]
```diff
@@ -1,15 +1,75 @@
...
+      async () => {
+        const downloadUrl = await uploadTask.snapshot.ref.getDownloadURL();
+        resolve(downloadUrl);
+      }
+    )
+  })
+}
+
+export async function updateBusinessInfo(uid, businessName, phoneNumber, businessLogo) {
+  const businessLogoDownloadUrl = await uploadFile(uid, 'images/logo', businessLogo);
```
Comment (eric-om) [inline src/firebase.js:64]: error handling would be another PR?

Comment (thetimothyp) [reply src/firebase.js:64]: Error handling is done by whoever is using this method so the UX can respond as necessary (e.g `BusinessInfo.js` needs to call `setIsLoading(false) when an error is caught, and the dashboard might want to pop up an error modal or something like that)

Comment (eric-om) [reply src/firebase.js:64]: Oops, meant to comment in the BusinessInfo.js file.

File: src/pages/BusinessInfo.js [hunk 2]
```diff
@@ -22,14 +29,18 @@ const BusinessInfo = () => {
...
+    onSubmit: async ({ businessName, phoneNumber, image }) => {
+      setIsLoading(true);
+
+      try {
+        // Submit business info to Firebase storage
+        await updateBusinessInfo(auth.user.uid, businessName, phoneNumber, image);
+        setIsLoading(false);
+        history.push('/dashboard');
+      } catch (e) {
+        console.error(e);
```
Comment (eric-om) [inline src/pages/BusinessInfo.js:41]: Here.

File: src/firebase.js (hunk 1 above)
Comment (eric-om) [reply src/firebase.js:64]: But also using `break` doesn't pass on the error, right?

Comment (thetimothyp) [reply src/firebase.js:64]: Oh yeah error UX will be separate.

> But also, does using break pass on the error?
It doesn't; I'll get rid of those breaks for now until they have their own error handling.

Comment (eric-om) [reply src/firebase.js:58]: I see. You can leave them if you want. I was just asking to better understand your process because you merge code more frequently.

**Review by eric-om — APPROVED** [review]
lgtm
